harvard-leeds-beckett-university.csl

```diff
@@ -15,6 +15,65 @@
     <updated>2015-03-23T12:34:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <macro name="sort-term">
+    <choose>
+      <if type="entry-dictionary">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+            </names>
+          </if>
+          <else>
+            <text macro="container-title"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="legal_case legislation" match="any">
+        <text macro="title"/>
+      </else-if>
+      <else-if type="bill">
+        <text macro="title"/>
+        <text macro="container-title" prefix=". " suffix=" Bill"/>
+      </else-if>
+      <else-if type="webpage">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+            </names>
+          </if>
+          <else-if variable="container-title">
+            <text macro="container-title"/>
+          </else-if>
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="song" variable="composer">
+        <names variable="composer">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+        </names>
+      </else-if>
+      <else-if variable="author">
+        <names variable="author">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+        </names>
+      </else-if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol" suffix=" ed."/>
+        </names>
+      </else-if>
```

Comment:
> If it's empty why should it exist?

It's only empty for "author", not for the substituting "editor".



> how is the "ed." suffix rendered in your simplified example?

Through `<label/>`. It calls the short form of the "editor" term as defined in the CSL locale file (or as redefined in the style itself).





> Is there any way to get it to show after each name e.g. "Gobo, G. ed. & Mauceri, S. ed."?

That's rather non-standard. I guess that's only possible with your original code.